tests/framework/utils/k8s_helper.go

```diff
@@ -568,17 +568,17 @@ func (k8sh *K8sHelper) GetEventsFromNamespace(namespace, testName, platformName
 	if events == "" {
 		return
 	}
-	file.WriteString(events)
+	file.WriteString(events) //nolint, we can safely supress error, it don't effect much
 }
 
 func (k8sh *K8sHelper) appendPodDescribe(file *os.File, namespace, name string) {
 	description := k8sh.getPodDescribe(namespace, name)
 	if description == "" {
 		return
 	}
-	writeHeader(file, fmt.Sprintf("Pod: %s\n", name))
-	file.WriteString(description)
-	file.WriteString("\n")
+	writeHeader(file, fmt.Sprintf("Pod: %s\n", name)) //nolint, we can safely supress error, it don't effect much
```

Comment:
Same comment can be made in rest of this file for nolint
```suggestion
	writeHeader(file, fmt.Sprintf("Pod: %s\n", name)) //nolint, ok to ignore this test logging
```

Reply:
Done.